app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java

```diff
@@ -196,7 +211,9 @@ public static void setSpeakerGain(String value, Context context) {
                 }
```

Comment:
There should actually be a break; here too! You can either add it in or I could submit a PR fixing that.